services/github/README.md

```diff
@@ -8,19 +8,18 @@ The GitHub service monitors all of the repositories associated with an organizat
 Listens for WebHooks and, if they are valid, forwards them to a pulse exchange.
 
 ### Handlers
-Listen for WebHook triggered pulse messages and attempts to schedule Taskcluster tasks for any events related to a repository which contains a `.taskcluster.yml` file.
+Listens for WebHook-triggered pulse messages and attempts to schedule Taskcluster tasks for any events related to a repository which contains a `.taskcluster.yml` file.
```

Comment:
It's plural - there are many handlers, not just one

Reply:
good call, fixed